client/src/components/Rating.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+// Styling imports
+import { RatingWrapper, OverviewWrapper, OverviewText, CategoryWrapper, RatingContainer} from './styles/staticStyles.style'
```

Comment:
Since this is more than 3 things eslint would want you to split this up over multiple lines like
`import {`
`  RatingWrapper,`
`  OverviewWrapper,`
`...`
`} from './styles/staticStyles.style';`
I think it's easier to read that way but nbd either way